WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -314,3 +315,46 @@ $venue-map-wrapper-height: 400px;
     background-color: inherit;
   }
 }
+
+table.table-condensed.admin-competitions {
+  font-size: 13px;
+
+  tr.admin-table-header {
+    background-color: #ccc;
+  }
+
+  td {
+    vertical-align: middle;
+  }
+
+  .admin-delegate {
+    width: 230px;
+  }
+
+  .admin-date {
+    width: 130px;
+    max-width: 130px;
+    text-align: center;
+    white-space: nowrap;
+  }
+
+  .admin-button {
+    width: 50px;
+  }
+}
+
+.alert-orange {
+  @extend .alert-warning;
+  background-color: $orange;
+}
+
+.alert-green {
+  @extend .alert-success;
+  background-color: $green;
+}
+
+.hide-div {
```

Comment:
`.hide-div` is a bit missleading, my first thought would be that it's hiden, even in the right context it's not always "hide" as it could be "show". It would be better to call it like `.toggle-body` or something similar.